### ISSUE TITLE
Pictures not found when "Exclude Subfolders" is selected

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#984]
Pictures not found (v2.6.5) when "Exclude Subfolders" option is selected, even though the displayed folder path is the folder with the pictures in it.

**Comment by markpenny** [comment]
Yes, we are able to replicate the issue. It will be fixed by the next release.

**Comment by davidc86** [comment]
Ah, okay. I came here to say this is not yet fixed in v2.6.8 (which came out 22 Oct), but I see where Mark just wrote on 22 Oct that it will be fixed in the "next release". So, I will wait for the next round. :)

**Comment by davidc86** [comment]
Okay, now it is too diligent in looking for pictures.  

![Image](https://github.com/user-attachments/assets/24dd4bf8-2ef1-459a-bc7c-f4bf67f1da52)

The picture it is showing is actually in a subfolder under "local" (local/figures), but I've selected the box to exclude any subfolders, yet it is still finding them. 
I assumed the purpose of the Exclude Subfolders is to make sure pictures are pulled from an exact location, so as not to accidentally pull low-res pictures or some other such scenario. 

Personally, it doesn't matter for our folder structures, but if the check box is there, then it is probably best if it works as described.

**Comment by davidc86** [comment]
Further testing: it's worse. I can change that "Picture Folder" to point to a different Paratext project (with no pictures in it), and PTXprint will still create the PDF file with the correct pictures in it and not report any missing pictures.

So maybe PTXprint is ignoring that whole line of settings?

**Comment by markpenny** [comment]
I think we need a discussion on how these settings/options interact with each other.

From what I understand, the current way it works is to (always) look in the current project's
(a) **figures**
and
(b) **local/figures**
folders. It also looks in:
(c) the **PTXprint picture repository** in case the image is found there. (e.g. "C:\Users\<username>\AppData\Local\SIL\ptxprint\imagesets\ccsamples\images\*")

Then, if the option is selected, is also looks in:
(d) the **specified folder** 

So far, so good. But note that it also looks in ALL the subfolders of all the abovementioned folders.

I think the problem is that the option "Exclude Subfolders" doesn't stop it (or perhaps _no longer_ stops it) looking in (a), (b) and (c). So when you point to some other Project's folder with no pictures in it; it still goes ahead and looks in (a), (b) and (c) to find what it is looking for.

My impression is that "Use other Location" should be rechristened to "Exclusively use folder" and do exactly that. And Exclude Subfolders should only apply to the specified folder, not to all of them.

What are your thoughts, @davidc86 ?

**Comment by davidc86** [comment]
Wow. Yeah, I had no idea it went through all of those folders. And we had no way of stopping it from looking in those.
So yes, I agree with your idea to say, "Exclusively use folder", and this would stop PTXprint from looking anywhere else.

**Comment by markpenny** [comment]
Perhaps we need a Radio button for that additional folder:
(o)  Exclusively use Folder:
(  )  Also Look in Folder:

**Comment by davidc86** [comment]
NOT fixed yet. 
IFF that option should limit the folders searched then: It doesn't actually need a name change, as "Use folder" kind of excludes any other folder. But the popup will need changing slightly to say it is exclusive.

IFF that option is to provide users a DIFFERENT folder to PTXprint to ALSO search (like the popup note implies), then the label in the dialog should be changed to say "**_Also_** use folder:"
Because that is how it currently works, right?
